src/Misc/externals.sh

```diff
@@ -7,7 +7,8 @@ L1_MODE=$4
 CONTAINER_URL=https://vstsagenttools.blob.core.windows.net/tools
 NODE_URL=https://nodejs.org/dist
 NODE_VERSION="6.17.1"
-NODE10_VERSION="10.21.0"
+NODE10_VERSION="10.22.1"
+NODE14_VERSION="14.11.0"
```

Comment:
Also, we are not shipping Node 14 anymore. Remove this line please :)

Reply:
@alex-peck 
I have now update the version to 10.23.0

Reply:
Can you delete the NODE14_VERSION entry? Otherwise we are good with this.